### DISCUSSION TITLE
Is there a way to run the PBS post install script if I have PBS just installed in main host?  It says incorrect script

### DISCUSSION THREAD

**Discussion by mattfox27** [tteck/Proxmox#1276] [General]
I just installed PBS right in the main VM but when I try to run PBS post install scrip it says VM detected wrong script

**Comment by tteck** [comment]
While I'm uncertain why someone would choose to install a Proxmox Backup Server in that manner, my answer to your question is no. The script identifies Proxmox VE, beings as that's the underlying system.
